docs/config/ping-federate-example.md

```diff
@@ -152,4 +120,52 @@ MIICnzCCAYcCBgFkqEAQCDANBgkqhkiG9w0BAQsFADATMREwDwYDVQQDDAhhbGZyZXNjbzA
 
 9. On the **Activation & Summary** tab tick the **Active** checkbox in the **Connection Status** row and **Save**. 
 
-Your new connection should now appear in the **SP Connections** list.
+10. Back on the identity service homepage, click **Manage all SP**.
+11. Your new connection should now appear in the **SP Connections** list. To the right of it click **Export metadata**.
+12. In the summary tab, click **Export**.
+
+
+### Identity Service Configuration
+
+* At the Identity Service homepage, navigage to **Identity providers**.
+* Click on **Add provider...**.
+* Click on **SAML v2.0**.
+* Edit the name of alias if needed. This will be the name provided at the log in. Such as saml-4 in the following screen shot:
```

Comment:
I'm wondering if this could be explained a little bit clearer so the user knows what's going to happen when he/she does this. Something like:

> Edit the alias name for this identity provider. This alias will be presented to users on the Identity Service log-in screen.

In this case I don't think you even need this screenshot, I'd be inclined to use them only when there's lots of information to cover (in this case all that matters is one button with this name).